bundles/org.openhab.binding.upnpcontrol/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -15,4 +15,26 @@
 				The folder will be created on first use when it does not exist.</description>
 		</parameter>
 	</config-description>
+
+	<discovery-methods>
+		<discovery-method>
+			<service-type>upnp</service-type>
+			<match-properties>
+				<match-property>
+					<name>deviceType</name>
+					<regex>MediaRenderer</regex>
+				</match-property>
+			</match-properties>
+		</discovery-method>
+		<discovery-method>
+			<service-type>upnp</service-type>
```

Comment:
You can remove discovery for MediaServer. Without a media renderer from the binding, it is useless. The media renderer on its own can be used as an audio sink. So proposing the binding if there only is a server makes no sense.

Reply:
Umm. In my home the server is mains powered, but the renderers are battery powered.

Reply:
Ok, fair enough.